site/app/templates/course/SetFolderReleaseForm.twig

```diff
@@ -0,0 +1,98 @@
+{% block title %}Set Folder Release{% endblock %}
+{% block body %}
+
+    <center> <b>Choose a time to release the contents of this folder:</b>
+    <br/>
+    <br/>
+    <input name="release_date" id="date_to_release" class="date_picker" type="text" size="26" value="2019-06-24 12:00:00" data-fP=" " onChange="getNewDateTime(this)"/>
+
+    </center>
+    <script>
+
+        function getNewDateTime(me){
+            var releaseDate = document.getElementById("date_to_release");
+            var folderPath = releaseDate.dataset.fP;
+
+            // pass filename to server to record the new date and time of the file to be released
+            var newDateTime = $(me).val();
+            //set the value to the new date and time
+            releaseDate.value = newDateTime;
+
+        }
+        function determineRelease(nDT) {
+            //get current date and time to determine what color the box is based
+            //on time you want released and the current date
+            var today = new Date();
+            //in order to compare correctly, need to add a zero to all single digits
+            if (today.getMonth() + 1 < 10) {
+                var month = '0' + (today.getMonth() + 1);
+            } else {
+                var month = (today.getMonth() + 1);
+            }
+            if (today.getDate() < 10) {
+                var day = '0' + today.getDate();
+            } else {
+                var day = today.getDate();
+            }
+            if (today.getHours() < 10) {
+                var hours = "0" + today.getHours();
+            } else {
+                var hours = today.getHours();
+            }
+            if (today.getMinutes() < 10) {
+                var mins = "0" + today.getMinutes();
+            } else {
+                var mins = today.getMinutes();
+            }
+            if (today.getSeconds() < 10) {
+                var secs = "0" + today.getSeconds();
+            } else {
+                var secs = today.getSeconds();
+            }
+            var date = today.getFullYear() + '-' + month + '-' + day;
+
+            var time = hours + ":" + mins + ":" + secs;
+            var currentDT = date + ' ' + time;
+
+            //will add a thrid color for never
+            //custom colors for the input box not on sheet for readability
+            if (nDT <= currentDT) {
+                console.log(currentDT);
+                console.log(nDT);
+                console.log("released");
+                return "#87db88";
+            } else {
+                console.log(currentDT);
+                console.log(nDT);
+                console.log("unreleased");
+                return "#ffeb54";
+            }
+        }
+        function confirm( ){
+            var directory = document.getElementById("submit_time");
+            var dirArr = JSON.parse(directory.dataset.indir);
+            var ID = directory.dataset.iden;
+
+            var releaseDate = document.getElementById("date_to_release");
```

Comment:
![Screenshot from 2019-06-29 19-43-27](https://user-images.githubusercontent.com/20266703/60383668-733ae900-9aa6-11e9-80fc-1326bfc268b7.png)
If I leave the input content empty, then all my files will have empty timestamp. I feel this is some sort of undefined behaviour.